### ISSUE TITLE
Upgrade reagent / re-frame, figwheel

### ISSUE THREAD

**Issue by markokocic** [tiensonqin/exponent-cljs-template#65]
The templaten is using outdated versions of reagent, re-frame and figwheel.
```
marko@hst~/src/tmp/ttt6 $ lein ancient
[binaryage/oops "0.6.1"] is available but we use "0.5.8"
[reagent "0.8.0"] is available but we use "0.7.0"
[re-frame "0.10.5"] is available but we use "0.9.3"
[figwheel-sidecar "0.5.15"] is available but we use "0.5.14"
[com.cemerick/piggieback "0.2.2"] is available but we use "0.2.1"
```
I tried bumping versions, but got different types of errors.
Has anyone made it work with latest versions?

**Comment by prestancedesign** [comment]
Hi @markokocic,

This issue can be closed since I have upgraded all dependencies in this PR https://github.com/seantempesta/expo-cljs-template/pull/81.

**Comment by seantempesta** [comment]
Fixed via #81.  Thanks again @PrestanceDesign!
